client/packages/invoices/src/OutboundShipment/api/hooks/index.ts

```diff
@@ -2,6 +2,9 @@ import { Utils } from './utils';
 import { Lines } from './line';
 import { Document } from './document';
 
+export { useOutboundItems } from './line/useOutboundItems';
+export { useOutboundLines } from './line/useOutboundLines';
```

Comment:
removed the hooks that depended on table state - and in the interest of slowly moving away from the useOutbound nested hook , just explicitly exporting these ones for use